examples/multi-callback-all/index.ts

```diff
@@ -0,0 +1,165 @@
+import * as azure from "@pulumi/azure";
+
+const resourceGroup = new azure.core.ResourceGroup("example", { location: azure.Locations.WestUS2 });
+
+// HTTP
+const httpFunc = new azure.appservice.HttpFunction("http", {
+    callback: async (context, request) => ({ status: 200, body: "Hello World" }),
+});
+
+// Fire on schedule
+const timerFunc = new azure.appservice.TimerFunction("timer", {
+    schedule: { minute: 0 },
+    callback: async (context, timer) => {
+        console.log(`Running at ${timer}`);
+    },
+});
+
+// Storage Queue
+const storageAccount = new azure.storage.Account("storage", {
+    resourceGroupName: resourceGroup.name,
+    accountReplicationType: "LRS",
+    accountTier: "Standard",
+});
+
+const queue = new azure.storage.Queue("queue", {
+    resourceGroupName: resourceGroup.name,
+    storageAccountName: storageAccount.name,
+});
+
+const storageQueueFunc = new azure.storage.QueueFunction("storage-queue", {
+    queue,
+    callback: async (context, msg) => {
+        console.log(`Received a message ${msg.toString()}`);
+    },
+});
```

Comment:
shoudl we consider makin it possible to get a QueueFunction directly from a Queue.  so like we have onEvent to make a FunctionApp, we could have getEventFunction to get this QueueFunction.  it would provide an easy way to discover this and to compse things easily.  Thoughts?

Reply:
Sounds promising! Seems non-invasive, so I'll give it a try in a separate PR.